bundles/org.openhab.binding.tapocontrol/README.md

```diff
@@ -43,21 +56,32 @@ This is used for device discovery and to create a handshake (cookie) to act with
 
 The thing has the following configuration parameters:
 
-| Parameter          | Description                                                          |
-|--------------------|----------------------------------------------------------------------|
-| username           | Username (eMail) of your Tapo-Cloud                                  |
-| password           | Password of your Tapo-Cloud                                          |
+| Parameter              | Description                                                                                         |
+|------------------------|-----------------------------------------------------------------------------------------------------|
+| username               | Username (eMail) of your Tapo-Cloud                                                                 |
+| password               | Password of your Tapo-Cloud                                                                         |
+| cloudDiscovery         | Use Cloud Discovery-Service to get all in Tapo-App registered devices. Includes DeviceName. IP-Address and Encryption has to set manually                     |
+| udpDiscovery           | Use UDP Discovery-Service to discover online devices in the local network. Includes Encryption and IP-Address. Results will be merged with cloud discovery    |
+| onlyLocalOnlineDevices | [advanced] Uses Cloud and UPD-Discovery to get more informations but will only discover online devices via UDP |
+| broadcastAddress       | [advanced] Set broadcast address to your local subnet  if you have problems with default address   |
+| discoveryInterval      | [advanced] Interval in minutes when a background device scan should be executed. Default is 60     |
 
-## Thing Configuration
 
-The thing needs to be configured with `ipAddress`.
+## Thing Configurations
 
-The thing has the following configuration parameters:
+WiFi-Things needs to be configured with `ipAddress`.
+RF-Things need a Hub (WiFi-Device) to operate. 
```

Comment:
```suggestion
RF-based things need a SmartHub(WiFi-Device) to operate. 
```

Not sure if it should be Hub or SmartHub, or Smarthub, but it this moment this description and the table below don;t have a consistent naming for the hub